server/base/src/main/java/org/apache/accumulo/server/constraints/MetadataConstraints.java

```diff
@@ -105,6 +105,12 @@ private static boolean isValidColumn(ColumnUpdate cu) {
     return lst;
   }
 
+  @Override
+  public List<Short> check(SystemEnvironment env, Mutation mutation) {
+    context = env.getServerContext();
```

Comment:
Could possibly not have a SystemConstraint and only have a SystemEnv.  This class could just cast Env to SystemEnv.

Reply:
Fixed with the latest commit